main.py

```diff
@@ -2650,5 +2650,30 @@ async def recieve_vote(request):
     await claim_reward(user, channeley, type)
     return web.Response(text="ok", status=200)
 
+@bot.tree.command(name="cat_fact", description="get a random cat fact")
+async def cat_fact(message: discord.Interaction):
+    
+    facts = [
+        "you love cats",
+        "cat bot is in 23,805+ servers",
+        "chocolate is bad for cats",
+        "cat",
+        "cats land on their feet",
+        "cats bring you mice/birds as a gift",
+        "cats are the best"
+    ]
+
+    # give a fact from the list or the API
+    if randint(0, 1) == 0:
+        await message.response.send_message(choice(facts))
+    else:
+        async with aiohttp.ClientSession() as session:
```

Comment:
maybe we should defer in case the api is slow?

Reply:
i dont know why its slow. theres no delay in the command

Reply:
im not saying its slow, but if the api takes longer than 3 seconds to respond it will show as an error on user side due to discord. that api is not in our control so it can freely go down or be slow at some point

Reply:
i have a JSON file that has a lot of those facts. maybe we can use that instead of the api to avoid the slow api and the possibility of it crashing